target_goal:
  # medicine: {pose: [1.67495292067, 0.311393855561, 0.0], orientation: [0.0, 0.0, -0.0348673749669, 0.999391948218], height: , pan_tilt: [,,], obstacle_id: , object_id: , gripper: }
  # pick: {pose: [18.6546487292, 20.2203282843, 0.0], orientation: [0.0, 0.0, 0.29566571728, 0.955291465274]}
  # pick: {pose: [18.8546487292, 20.4203282843, 0.0], orientation: [0.0, 0.0, 0.29566571728, 0.955291465274]}
  pick: {pose: [24.1027336121, -34.6507339478, 0.0], orientation: [0.0, 0.0, 0.912737779168, 0.912737779168]}
  #place: {pose: [15.4677233399, 18.1708953986, 0.0], orientation: [0.0, 0.0, -0.971412861571, 0.237396403459]}
  place: {pose: [24.1236400604, -27.3544998169, 0.0], orientation: [0.0, 0.0, 0.90442983562, -0.426622400304]}
  #standby: {pose: [16.9687800412, 18.0345954985, 0.0], orientation: [0.0, 0.0, 0.900585753523, 0.434678387491]}
  standby: {pose: [19.8948974609, -30.120262146, 0.0], orientation: [0.0, 0.0, -0.338870259957, 0.940833113212]}

arm_pose:
  standby: [1.708201856921426, 2.451008085005237, 5.591980488377151, 2.1399074601582133, 4.669752303964739, -0.02545106487907388]
  # standby: [1.6711630119547674, 2.4111349708484613, 5.551345443832504, 1.8526619676175207, 4.754093017790894, 0.002003161892558725]
  pre_place: [1.708201856921426, 2.451008085005237, 5.591980488377151, 2.1399074601582133, 4.669752303964739, -0.02545106487907388]

box_information:
  # count: 8
  box_1: {arm_pose: [-1.4158301787564938, 1.7658556992541696, 5.130710867563284, -1.4283462369315392, 4.6123297571726685, 6.0551569325315135], height: 0.0, size: 0.6, id: 88}
  box_2: {arm_pose: [0.6925838036148787, 1.5310441201516107, 5.3860877760775985, 1.0155708487762254, 5.031722235109475, 6.812369264216], height: 0.0, size: 0.6, id: 88}
  box_3: {arm_pose: [1.4200486259231793, 1.5627822092845756, 5.115967607768606, 1.4443169476726354, 4.750679377905631, 6.671243031730521], height: 0.0, size: 0.6, id: 88}
  box_4: {arm_pose: [-1.4246795967427182, 2.687689602667195, 5.335948974866895, -1.4181394057377015, 4.656979783376719, 6.77784457647643], height: 0.0, size: 0.6, id: 88}
  box_5: {arm_pose: [0.2131278884805384, 3.075370756239992, 5.953781575108776, 0.2569129532520448, 4.3605462519171985, 6.183140997669669], height: 0.0, size: 0.6, id: 273}
  box_6: {arm_pose: [1.40291238273101, 2.7449496966327707, 5.465554905108909, 1.4526592997873087, 4.623533143246414, 5.844537109297071], height: 0.0, size: 0.6, id: 273}
  box_7: {arm_pose: [-1.4246795967427182, 2.687689602667195, 5.335948974866895, -1.4181394057377015, 4.656979783376719, 6.77784457647643], height: 0.4, size: 0.6, id: 88}
  box_8: {arm_pose: [0.2131278884805384, 3.075370756239992, 5.953781575108776, 0.2569129532520448, 4.3605462519171985, 6.183140997669669], height: 0.4, size: 0.6, id: 88}
  box_9: {arm_pose: [1.40291238273101, 2.7449496966327707, 5.465554905108909, 1.4526592997873087, 4.623533143246414, 5.844537109297071], height: 0.4, size: 0.6, id: 273}
